api/actions/createWorkItem.js

```diff
@@ -23,10 +27,14 @@ module.exports = {
     eventName: GithubEvents.events.ISSUE_LABELED,
     fn: async function (req) {
         const {
-            issue: { labels, url, title, body, milestone }
+            issue: { labels, url, body, milestone }
+        } = req.body;
+        var {
```

Comment:
use `let` here instead of `var` or I will rather compute a new variable like `normalizedTitle` and avoid doing mutation